src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeReportToFile(report, toFileName);
+    }
+
+    public static String readFile(String fromFileName) {
+        int supply = 0;
+        int buy = 0;
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            String value = reader.readLine();
+
+            while (value != null) {
+                String[] valueArr = value.split(",");
+
+                if (valueArr[0].equals("supply")) {
+                    supply += Integer.parseInt(valueArr[1]);
+                } else {
+                    buy += Integer.parseInt(valueArr[1]);
```

Comment:
In this method you need only read, not parse or any operation

Reply:
done